javacord-api/src/main/java/org/javacord/api/entity/user/UserFlag.java

```diff
@@ -0,0 +1,50 @@
+package org.javacord.api.entity.user;
+
+public enum UserFlag {
+
+    DISCORD_EMPLOYEE(1, "Discord Employee"),
+    DISCORD_PARTNER(2, "Discord Partner"),
+    HYPESQUAD_EVENTS(4, "HypeSquad Events"),
+    BUG_HUNTER_LEVEL_1(8, "Bug Hunter Level 1"),
+    HOUSE_BRAVERY(64, "House Bravery"),
```

Comment:
I think it would be better to write it like in the Discord docs instead of using decimal notation:
```suggestion
    DISCORD_EMPLOYEE(1 << 0, "Discord Employee"),
    DISCORD_PARTNER(1 << 1, "Discord Partner"),
    HYPESQUAD_EVENTS(1 << 2, "HypeSquad Events"),
    BUG_HUNTER_LEVEL_1(1 << 3, "Bug Hunter Level 1"),
    HOUSE_BRAVERY(1 << 5, "House Bravery"),
```
and so on